detekt-generator/build.gradle.kts

```
@@ -86,6 +83,36 @@ val generateDocumentation by tasks.registering(JavaExec::class) {
     )
 }
 
+val generatedLibrariesConfig: Configuration by configurations.creating {
+    isCanBeConsumed = true
+    isCanBeResolved = false
+}
+
+val generatedRulesConfig: Configuration by configurations.creating {
+    isCanBeConsumed = true
+    isCanBeResolved = false
+}
+
+val generatedCoreConfig: Configuration by configurations.creating {
+    isCanBeConsumed = true
+    isCanBeResolved = false
+}
+
+artifacts {
+    add(generatedLibrariesConfig.name, file(librariesConfigFile)) {
+        builtBy(generateDocumentation)
+    }
+    add(generatedRulesConfig.name, file(ruleauthorsConfigFile)) {
```

would you mind renaming this to
```suggestion
    add(generatedRuleauthorsConfig.name, file(ruleauthorsConfigFile)) {
```
"rulesconfig" sounds like it's all the rules.

---

done 👍